src/displayapp/screens/WatchFaceTerminal.cpp

```diff
@@ -0,0 +1,264 @@
+    auto dayOfWeek = static_cast<Pinetime::Controllers::DateTime::Days>(date::weekday(yearMonthDay).iso_encoding());
+
+    auto hour = time.hours().count();
+    auto minute = time.minutes().count();
+	  auto second = time.seconds().count();
+
+    char minutesChar[6];
+    sprintf(minutesChar, "%02d", static_cast<int>(minute));
+
+    char hoursChar[8];
+
+    char ampmChar[3];
+	    if (settingsController.GetClockType() == Controllers::Settings::ClockType::H24) {
+        sprintf(hoursChar, "%02d", hour);
+      } else {
+        if (hour == 0 && hour != 12) {
+          hour = 12;
+          sprintf(ampmChar, "AM");
+        } else if (hour == 12 && hour != 0) {
+          hour = 12;
+          sprintf(ampmChar, "PM");
+        } else if (hour < 12 && hour != 0) {
+          sprintf(ampmChar, "AM");
+        } else if (hour > 12 && hour != 0) {
+          hour = hour - 12;
+          sprintf(ampmChar, "PM");
+        }
+        sprintf(hoursChar, "%02d", hour);
+    }
+
+	char secondsChar[5];
+    sprintf(secondsChar, "%02d", static_cast<int>(second));
+
+	auto batteryValue = static_cast<uint8_t>(batteryController.PercentRemaining());
+/*
+	char batteryGauge[12];
+    if (batteryValue > 75){
+      sprintf(batteryGauge, "[++++]");  //100-75
+    }
+    else if (batteryValue > 50){
+      sprintf(batteryGauge, "[+++.]");  //75-50
+    }
+    else if (batteryValue > 25){
+      sprintf(batteryGauge, "[++..]");  //50-25
+    }
+    else if (batteryValue > 10){
+      sprintf(batteryGauge, "[+....]"); //25-10
+    }
+    else if (batteryValue > 0){
+      sprintf(batteryGauge, "[!!!!!]"); //10-0
+    }
+*/
+    char battStr[24];
+    sprintf(battStr, "[BATT]#387b54 %d%\%#", batteryValue);
+    lv_label_set_text(batteryPercent, battStr);
+
+    if(hoursChar[0] != displayedChar[0] ||
+       hoursChar[1] != displayedChar[1] ||
+       minutesChar[0] != displayedChar[2] ||
+       minutesChar[1] != displayedChar[3] ||
+       secondsChar[0] != displayedChar[4] ||
+       secondsChar[1] != displayedChar[5])
+       {
+      displayedChar[0] = hoursChar[0];
+      displayedChar[1] = hoursChar[1];
+      displayedChar[2] = minutesChar[0];
+      displayedChar[3] = minutesChar[1];
+      displayedChar[4] = secondsChar[0];
+      displayedChar[5] = secondsChar[1];
+
+      if (settingsController.GetClockType() == Controllers::Settings::ClockType::H12) {
+        if (hoursChar[0] == '0') {
+          hoursChar[0] = ' ';
+        }
+      }
+
+      char timeStr[42];
+      sprintf(timeStr, "[TIME]#11cc55 %c%c:%c%c:%c%c %s#", hoursChar[0],hoursChar[1],minutesChar[0], minutesChar[1], secondsChar[0], secondsChar[1], ampmChar);
+
+      lv_label_set_text(label_time, timeStr);
+    }
+
+    if ((year != currentYear) || (month != currentMonth) || (dayOfWeek != currentDayOfWeek) || (day != currentDay)) {
+
+	    char dateStr[38];
+      sprintf(dateStr, "[DATE]#007fff %04d.%02d.%02d#", short(year), char(month), char(day));
```

Comment:
suggesting to use `-` instead of `.` for the iso-date-string. That would be the ISO 8601 format that [pythons date.isoformat()](https://docs.python.org/3/library/datetime.html#datetime.date.isoformat) returns
```suggestion
      sprintf(dateStr, "[DATE]#007fff %04d-%02d-%02d#", short(year), char(month), char(day));
```

Reply:
no, please, no...
it will not look so aesthetically pleasing and authentic